superset-frontend/src/components/Chart/Chart.jsx

```diff
@@ -288,6 +288,23 @@ class Chart extends React.PureComponent {
       );
     }
 
+    if (
+      !isLoading &&
+      !chartAlert &&
+      isFaded &&
+      ensureIsArray(queriesResponse).length === 0
+    ) {
+      return (
+        <EmptyStateBig
+          title={t('Your chart is ready to go!')}
+          description={t(
+            'Click on "Create chart" button in the control panel on the left to preview a visualisation',
```

Comment:
@rusackas Help me out here but I think "visualiZation" is more generally accepted in the US right?

Reply:
In the US, yes. We're weird. We also optimize and scrutinize and so forth. Maybe one day we'll have American English and English localizations fleshed out :D